astronomer/providers/http/sensors/http.py

```diff
@@ -42,7 +24,8 @@ def response_check(response, task_instance):
     :type request_params: a dictionary of string key/value pairs
     :param headers: The HTTP headers to be added to the GET request
     :type headers: a dictionary of string key/value pairs
-    :param response_check: A check against the 'requests' response object.
+    :param response_check: This param is currently not supported.
```

Comment:
Should we remove this param from the docstring here altogether since we're not supporting it?

Reply:
Yes, agreed with @pankajkoti  can we remove this?